html2docx.py

```diff
@@ -54,9 +62,16 @@ def _reset(self) -> None:
 
         # Formatting options
         self.pre = False
+        self.alignment: Optional[int] = None
         self.attrs: List[List[str]] = []
         self.collapse_space = True
 
+    def init_p(self, attrs: List[Tuple[str, Optional[str]]]) -> None:
+        for attr in attrs:
+            for name, value in attr_to_css(attr):
+                if name == "text-align":
+                    self.alignment = ALIGNMENTS[value]
```

Comment:
`text-align` offers other valid values (`justify-all`, `start`, `end`, etc.). An unsupported property will make the document generation fail. Perhaps only known values should be interpreted, i.e. `self.alignment = ALIGNMENTS.get(value)`?